cabal.project

```diff
@@ -21,7 +21,7 @@ tests: true
 benchmarks: true
 
 -- The only sensible test display option
-test-show-details: streaming
+test-show-details: direct
```

Comment:
Does it? In what way? I don't mind taking a change for this, I'd just like some justification rather than a random diff in the middle of another PR :)

Reply:
`streaming` doesn't colourize output, and `tasty` (or at least, some of its runners) provide colourized diffs. It also gives (using _any_ runner) a clear indication of passed tests (which are green) and failed ones (which are red).

Reply:
Hmmm.  I'm sure I used to see colourised test results, but indeed with `streaming` I don't. The `streaming` option seems to have been there for about six months though.

Reply:
Okay, good to know. Can you add a comment saying "this is the only mode that allows colorized output from tasty"